src/Components/Hooks/useFetch.jsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from 'react';
+import fetchQuestions from '../../Api';
+
+export default function useFetch() {

```

Comment:
nice approach. But the name `useFetch` is too vague to be tied to a single HTTP method and endpoint. I will suggest something like `useQuestions`

Reply:
Ok sir i consider